assets/js/blocks/mini-cart/mini-cart-contents/inner-blocks/mini-cart-shopping-button-block/block.tsx

```diff
@@ -23,15 +24,16 @@ const Block = ( {
 	}
 
 	return (
-		<div
-			className={ classNames(
-				className,
-				'wc-block-mini-cart__shopping-button'
-			) }
-		>
-			<a href={ SHOP_URL }>
+		<div className="wp-block-button aligncenter">
```

Comment:
I think the `aligncenter` class name has to be changed to `has-text-align-center` for it to actually center.

Reply:
You are right, fixes, thanks!